tests/unit/s2n_cbc_verify_test.c

```diff
@@ -123,6 +125,7 @@ int main(int argc, char **argv)
     struct s2n_blob r = {.data = random_data, .size = sizeof(random_data)};
 
     BEGIN_TEST();
+#if defined(__x86_64__) || defined(__i386__)
```

Comment:
Leave a comment describing why this is disabled.

We should probably also ifdef out the function locals to avoid unused variable warnings.